### ISSUE TITLE
Custom Scalar Types : CustomType not generated

### ISSUE THREAD

**Issue by Joeluck** [apollographql/apollo-android#1012]
I tried to add a custom type, I duplicated the README.txt code : 
apollo {
  customTypeMapping = [
    "Date" : "java.util.Date"
  ]
}

But the CustomType.DATE is not generated. 

Am I missing something ? 

No warning or issue during the build. 

Currently using v1.0.0-alpha

**Comment by sav007** [comment]
could you please try snapshot version, we are going to release it soon

**Comment by saurabhdtu** [comment]
Has this been fixed?
I am using this in gradle, but I am not able to see any custom scalar type getting generated besides ID.
```
apollo {
    customTypeMapping = [
            "DateTime" :"java.lang.String"
    ]
}
```

**Comment by Nazacheres** [comment]
Why the issue is closed? Have exactly the same issue with version 1.0.0 :(

**Comment by saurabhdtu** [comment]
I had the same issue. But I had some issue in the  schema json file due to which the CustomType weren't getting generated. Make sure your schema is properly defined with CustomType. For eg. the ID scalar is automatically accepted as String and is generated in CustomType enum. In the same way your  other scalars will be generated which you can then translate to your custom type using the adapter

**Comment by saurabhdtu** [comment]
If it doesn't get generated even after that, manually delete the generated enum CustomType and rebuild the project again.

**Comment by Nazacheres** [comment]
@saurabhdtu Thanks, you were right, I didn't have those fields that i wanted to map in my schema. Some error or warning would be handy tho.

**Comment by saurabhdtu** [comment]
@Nazacheres Yes, debugging graphql errors is not simple with android studio errors. So I would recommend you a simple way to validate your graphql queries in a simpler way on the api documentation web page: 
www.<host-domain>/graphql
Copy and paste your queries here so that if there is any error in your queries, they'll be highlighted over there.
